Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -2300,6 +2335,15 @@ - (UIEdgeInsets)currentlyVisibleContentInsetsWithScale:(CGFloat)scaleFactor obsc
     return contentInsets;
 }
 
+- (BOOL)_shouldDeferGeometryUpdates
+{
+    if (_dynamicViewportUpdateMode != WebKit::DynamicViewportUpdateMode::NotResizing)
+        return YES;
+    if (_liveResizeParameters)
+        return YES;
+    return NO;
```

Comment:
Nit - would this be clearer as just `_liveResizeParameters || _dynamicViewportUpdateMode != WebKit::DynamicViewportUpdateMode::NotResizing`?

Reply:
Sure :)